vendor.conf

```diff
@@ -17,8 +17,8 @@ github.com/docker/go-units                          v0.4.0
 github.com/godbus/dbus/v5                           v5.0.3
 github.com/gogo/googleapis                          v1.3.2
 github.com/gogo/protobuf                            v1.3.1
-github.com/golang/protobuf                          v1.3.5
-github.com/google/go-cmp                            v0.2.0
+github.com/golang/protobuf                          v1.4.2
```

Comment:
I updated protobuf deps to match runc go.mod, but probably not necessarily needed

Reply:
Do we need to do it separate so that the changes can be reviewed separately? I see there's only minor changes in runc itself (related to https://github.com/opencontainers/runc/pull/2520)

Reply:
@AkihiroSuda the protobuf, i'd prefer to avoid it given the churn it will cause in both cadvisor and kubernetes at this last minute in kubernetes release cycle.

Reply:
updated to drop protobuf stuff